pdns/dnsdist.hh

```diff
@@ -51,6 +54,95 @@ extern uint16_t g_ECSSourcePrefixV4;
 extern uint16_t g_ECSSourcePrefixV6;
 extern bool g_ECSOverride;
 
+
+class QTag
+{
+
+public:
+
+QTag()
+{
+}
+
+~QTag()
+{
+}
+
+bool add(std::string strLabel, std::string strValue)
+{
+bool bStatus = true;
+
+   tagData.insert( {strLabel, strValue});
+  return(bStatus);
```

Comment:
It looks like it always return `true`, so perhaps the return value might be omitted?